enclave/core/CMakeLists.txt

```diff
@@ -330,6 +359,10 @@ if(USE_DEBUG_MALLOC)
     message("USE_DEBUG_MALLOC is set, building oecore with memory leak detection.")
 endif()
 
+if (COMPILE_SYSTEM_EDL)
```

Comment:
Since 267 differs in style from 362, please run format-code on this file.  If format-code isn't processing CMakeLists.txt files, then please update issue #2797 with a comment about this being another gap.

Reply:
It only runs on .c, .cpp, and .h. I have updated the linked issue